src/SqlPersistence/Subscription/SubscriptionCommandBuilder.cs

```diff
@@ -29,7 +29,7 @@ public static SubscriptionCommands Build(SqlVariant sqlVariant, string tablePref
                     break;
 
                 case SqlVariant.Oracle:
-                    tableName = $"{tablePrefix.ToUpper()}SS";
+                    tableName = string.IsNullOrEmpty(schema) ? $"\"{tablePrefix.ToUpper()}SS\"" : $"\"{schema}\".\"{tablePrefix.ToUpper()}SS\"";
```

Comment:
Here we don't call schema.ToUpper but we do in https://github.com/Particular/NServiceBus.Persistence.Sql/pull/202/commits/0ba012f075cc64a60bf9574525faa826f2cf0259#diff-b4d07e6bfe521906a2c801492dd4a453R85, seems like a lack of consistency? not sure if it's ok to always uppercase schemas which may be custom and created without our scripts.